src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -62,7 +62,6 @@
                 <NotebookEmbed v-for="embed in entry.embeds"
                                :key="embed.id"
                                :embed="embed"
-                               :entry="entry"
                                @removeEmbed="removeEmbed"
```

Comment:
Removing embeds, entries, pages or sections doesn't seem to delete the linked images. They should be deleted if they're not used or its just noise.

Reply:
yes I think we discussed this issue in one of our stand up, for now I think decided to leave those objects (as ObjectAPI.prototype.delete is not implemented). @akhenry is that correct?

Reply:
I think we should delete them. 

We did discuss leaving them because solving deletion generally for objects is too hard right now (because it's impossible to know whether there are other references to an object). 

Notebooks, however, are a special case. I think we can confidently delete objects in this case. The only case where there might be multiple references to a single image is if an entry is duplicated. We can solve this trivially by duplicating the image when an entry is duplicated, thereby enforcing a 1:1 relationship between entries and images. This is not so crazy, the duplicate action does the same thing with objects.

Reply:
@nikhilmandlik  Any progress on this?